pkg/settings/setting.go

```diff
@@ -129,12 +125,33 @@ var (
 
 	Rke2DefaultVersion = NewSetting("rke2-default-version", "")
 	K3sDefaultVersion  = NewSetting("k3s-default-version", "")
+
+	// KubeconfigGenerateToken determines whether the UI will return a generate token with kubeconfigs.
+	// If set to false the kubeconfig will contain a command to login to Rancher.
+	KubeconfigGenerateToken = NewSetting("kubeconfig-generate-token", "true")
+
+	// Deprecated: On removal use kubeconfig-default-ttl-minutes for all kubeconfigs.
+	KubeconfigTokenTTLMinutes = NewSetting("kubeconfig-token-ttl-minutes", "960") // 16 hours
```

Comment:
We should still say that this setting applies only when `kubeconfig-generate-token` is set to false, or when user's use the CLI token command. Also that this will override KubeconfigDefaultTokenTTLMinutes (pretty sure that part is true).

Reply:
They are separate due to the fact that KubeconfigDefaultTokenTTLMinutes will not be used for CLI token command until this deprecated setting is removed. That being said I agree we should explicitly state that `KubeconfigTokenTTLMinutes` takes president for tokens created through the CLI or when `kubeconfig-generated-token` is set to false.